circuit_maintenance_parser/data.py

```diff
@@ -0,0 +1,47 @@
+"""Definition of Data classes."""
+import logging
+
+from collections import namedtuple
+from typing import List
+
+import email
+from pydantic import BaseModel, Extra
+
+
+logger = logging.getLogger(__name__)
+
+
+DataPart = namedtuple("DataPart", "type content")
+
+
+class NotificationData(BaseModel, extra=Extra.forbid):
+    """Base class for Notificaiton Data types."""
```

Comment:
Can this docstring please be detailed a bit more? Maybe with examples of what is meant by "notification data", how this can store one or several DataParts, etc.?

Reply:
sure, this PR is not finished (documentation, tests, etc.). I will take this into account after the initial review of the design